openedx/adg/lms/webinars/helpers.py

```diff
@@ -148,17 +150,18 @@ def schedule_webinar_reminders(user_emails, email_context):
             user_emails,
             email_context,
             webinar_start_time - timedelta(days=7),
+            save_mandrill_msg_ids=True
         )
 
 
-def save_scheduled_reminder_ids(mandrill_response, template_name, webinar_id):
+def save_scheduled_reminder_ids(mandrill_response, template_name, webinar_reminders_context):
```

Comment:
Is there any specific reason we have decided to use `webinar_reminders_context` dict instead of `webinar_id`? Aren't we simply extracting the id from the dict later on?

Reply:
Thank you for the detailed explanation. I get it. Good work!